packages/pci-sdk-iframe/src/pages/app/components/InlineForm/InlineForm.tsx

```diff
@@ -17,13 +17,13 @@ export default function InlineForm(props: IInlineFormProps) {
 	return (
 		<form
 			className="InlineForm"
-			style={props.theme.inlineForm}
+			style={props.theme.inlineForm?.container}
```

Comment:
what happens if we don't pass an inlineForm in the theme. Does this throws? We had a similar issue in a previous version 🤔

Reply:
This just doesn’t add the class to the form. In that case, if you don't extend any of our themes the border is going to be the default border of the button.